recipes-smalltalk/smalltalk/smalltalk-package.inc

```diff
@@ -1,17 +1,2 @@
-DEPENDS = "smalltalk"
-
-INC_PR="r4"
-
-do_compile() {
-	gst < /dev/null
-	gst-package --target-directory=. package.xml
-}
-
-do_install () {
-	install -d ${D}${datadir}/smalltalk
-
-	install -m 0644 ${S}/*.star ${D}${datadir}/smalltalk
-} 
-
-FILES_${PN} += "${datadir}/smalltalk"
-
+inherit smalltalk_package
+INC_PR = "r5"
```

Comment:
This variable is dead and I artificially kept it alive for us to work with $OLD version. Not sure if you need to upgrade it.